Suvorov.LNU.TwitterClone.Models/Frontend/CreateUserRequest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Suvorov.LNU.TwitterClone.Models.Database;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Suvorov.LNU.TwitterClone.Models.Frontend
+{
+    public class CreateUserRequest
+    {
+        [Required(ErrorMessage = "Who are you?")]
+        [StringLength(50)]
+        public string? Name { get; set; }
+
+        [Required(ErrorMessage = "Username is required.")]
+        [StringLength(50)]
+        public string? UserName { get; set; }
+
+        [Required(ErrorMessage = "Email is required.")]
+        [EmailAddress(ErrorMessage = "Enter valid email address.")]
+        public string? EmailAddress { get; set; }
+
+        [Required]
+        [DataType(DataType.Date)] 
+        public DateTime Birthday { get; set;}
+
+        [Required(ErrorMessage = "Password is required.")]
+        [RegularExpression("^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[a-zA-Z\\d]{8,}$", 
```

Comment:
Add more error info, I entered `!` and still got the same message

Reply:
kk, will add more errors handling